DABI.js

```diff
@@ -0,0 +1,482 @@
+					},
+					{
+						"tag": "Krankenversorgung"
+					},
+					{
+						"tag": "Leipzig"
+					},
+					{
+						"tag": "Medizin"
+					},
+					{
+						"tag": "Medizinbibliothek"
+					},
+					{
+						"tag": "Multicenter-Studie"
+					},
+					{
+						"tag": "Münster"
+					},
+					{
+						"tag": "Regensburg"
+					},
+					{
+						"tag": "Umfrage"
+					},
+					{
+						"tag": "Universität Freiburg"
+					},
+					{
+						"tag": "Universität Leipzig"
+					},
+					{
+						"tag": "Universität Münster"
+					},
+					{
+						"tag": "Universität Regensburg"
+					}
+				],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "http://dabi.ib.hu-berlin.de/cgi-bin/dabi/vollanzeige.pl?artikel_id=21283&modus=html",
+		"items": [
+			{
+				"itemType": "journalArticle",
+				"title": "\"Was ihr wollt!\" Nutzungsgesteuerter Einkauf von Medien an der Staatsbibliothek zu Berlin",
+				"creators": [
+					{
+						"firstName": "Janin",
+						"lastName": "Taubert",
+						"creatorType": "author"
+					}
+				],
+				"date": "2014",
+				"issue": "3",
+				"libraryCatalog": "DABI",
+				"pages": "79-81",
+				"publicationTitle": "Bibliotheks-Magazin",
+				"shortTitle": "Was ihr wollt",
+				"url": "http://staatsbibliothek-berlin.de/fileadmin/user_upload/zentrale_Seiten/ueber_uns/pdf/Bibliotheksmagazin/Bibliotheksmagazin_3-2014.pdf",
+				"volume": "9",
+				"attachments": [
+					{
+						"title": "DABI Full Text PDF",
+						"mimeType": "application/pdf"
+					}
+				],
+				"tags": [
+					{
+						"tag": "Benutzerorientierter Bestandsaufbau"
+					},
+					{
+						"tag": "Benutzerorientierung"
+					},
+					{
+						"tag": "Berlin"
+					},
+					{
+						"tag": "Bestand"
+					},
+					{
+						"tag": "Bestandsaufbau"
+					},
+					{
+						"tag": "Bibliothekswesen"
+					},
+					{
+						"tag": "Demand Driven Acquisition (DDA)"
+					},
+					{
+						"tag": "E-Book"
+					},
+					{
+						"tag": "Evidence Based Selection (EBS)"
+					},
+					{
+						"tag": "Kundenorientierter Bestandsaufbau"
+					},
+					{
+						"tag": "Patron Driven Acquisition (PDA)"
+					},
+					{
+						"tag": "Purchase On Demand (POD)"
+					},
+					{
+						"tag": "Staatsbibliothek zu Berlin - Preußischer Kulturbesitz (SBB PK)"
+					}
+				],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "http://dabi.ib.hu-berlin.de/cgi-bin/dabi/vollanzeige.pl?artikel_id=5676&modus=html",
+		"items": [
+			{
+				"itemType": "journalArticle",
+				"title": "Anpassung der Personalstruktur der Fachhochschulbibliotheken in Nordrhein-Westfalen an die Erfordernisse der neunziger Jahre",
+				"creators": [],
+				"date": "1992",
+				"issue": "1",
+				"libraryCatalog": "DABI",
+				"pages": "364-372",
+				"publicationTitle": "Mitteilungsblatt des Verbandes der Bibliotheken des Landes Nordrhein-Westfalen",
+				"shortTitle": "Anpassung der Personalstruktur der Fachhochschulbibliotheken in",
```

Comment:
This suggests for me the short-title code isn't good. I'd be inclined to let Zotero handle them automatically.

Reply:
From the code I see, that the short title consists of at most 6 words.

Reply:
Yeah, I don't think that's acceptable. It'll too often produce nonsense such as this.

Reply:
How does Zotero handle short titles automatically? I'd be glad to drop this piece of code since I'm not happy with it either. Should I just do and accept the PR proposed by @zuphilip?

Reply:
Zotero just splits before colons, exclamation marks, and question marks, which I think works well here (doesn't always in Germany, where we also see subtitles set off by periods). 

> Should I just do and accept the PR proposed by @zuphilip?

yes, please

Reply:
The PR fixes some other things and therefore it is good to accept it anyways. Then, afterwards, we need to delete the lines dealing with shortTitle, and update the tests cases.

Reply:
How do I do it? Just say yes or click on some button (e.g. "Close pull request"?). In case you need me to say something so, yes.

Reply:
Click on the green button "Create a merge commit" or "Squash and merge" here: https://github.com/jmiba/translators/pull/1

Reply:
Thank you for your help correcting and cleaning up the code. Made me learn something.

Reply:
Okay, that worked. Now maybe `git pull <your-fork>...` and then delete the lines for the short title and update the test cases (use Scaffold). Can you do that?

Reply:
OK, done (I think).